### ISSUE TITLE
Cannot read property 'protocolFamily'

### ISSUE THREAD

**Issue by prvjhamada** [fido-alliance/conformance-tools-issues#227]
## What protocol and version of the protocol are you testing?

FIDO2

## What is your implementation class?

Server

## What is the version of the tool are you using?

v0.9.420

## Issue description
On Server-ServerAuthenticatorAttestationResponse-Resp-10,the following error occured on P-10 and P-11 tests.
```
TypeError: Cannot read property 'protocolFamily' of undefined
```
It seems like the tests fails to load right metadata statement.
I think the tests should load following ones.

- P-10
`Virtual Secp384r1 SHA384 FIDO2 Conformance Testing CTAP2 Authenticator`

- P-11
`Virtual Secp521r1 SHA512 FIDO2 Conformance Testing CTAP2 Authenticator`

**Comment by herrjemand** [comment]
You are correct. Resolved in v0.9.421

**Comment by herrjemand** [comment]
Thanks @prvjhamada
